code/game/objects/items/storage/bags.dm

```diff
@@ -391,6 +391,11 @@
 	STR.insert_preposition = "in"
 	STR.can_hold = typecacheof(list(/obj/item/slime_extract, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/glass/beaker, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/blood, /obj/item/reagent_containers/hypospray/medipen, /obj/item/reagent_containers/food/snacks/deadmouse, /obj/item/reagent_containers/food/snacks/monkeycube, /obj/item/organ, /obj/item/bodypart))
 
+/obj/item/storage/bag/bio/pre_attack(atom/A, mob/living/user, params)
+	if(SEND_SIGNAL(src, COMSIG_ITEM_PRE_ATTACK, A, user, params) & COMPONENT_NO_ATTACK & !istype(A, /obj/item/slimecross/reproductive))
```

Comment:
This isn't right, you don't use single & for bool checks

Reply:
Right no shortcircuit behavior